XIVSlothCombo/Combos/MCH.cs

```diff
@@ -110,16 +110,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                 if (IsEnabled(CustomComboPreset.MachinistAlternateMainCombo))
                 {
-                    if (reassembleCD.CooldownRemaining >= 55 && !airAnchorCD.IsCooldown && level >= 76)
+                    if (!airAnchorCD.IsCooldown && level >= 76)
                         return MCH.AirAnchor;
                     if (reassembleCD.CooldownRemaining >= 55 && !drillCD.IsCooldown && level >= 58)
                         return MCH.Drill;
-                    if (reassembleCD.CooldownRemaining >= 55 && !hotshotCD.IsCooldown && level <= 75)
+                    if (!hotshotCD.IsCooldown && level <= 75)
                         return MCH.HotShot;
-                    else
                     if (level >= 84)
                     {
-                        if (reassembleCD.IsCooldown && !airAnchorCD.IsCooldown)
```

Comment:
Why would you remove the check to see if Reasemble is on cooldown and have AirAnchor be used willynilly it makes no sence.